discord/dockerfiles/java-Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM openjdk:17
```

Comment:
Does there happen to be an Alpine container for OpenJDK 17?

Reply:
Not for OpenJDK 17. AFAIK, Alpine containers only exist for the JDK early access builds. (eg. JDK 19)